reference-implementation/lib/ReadableByteStreamController-impl.js

```diff
@@ -48,6 +49,9 @@ exports.implementation = class ReadableByteStreamControllerImpl {
     if (chunk.buffer.byteLength === 0) {
       throw new TypeError('chunk\'s buffer must have non-zero byteLength');
     }
+    if (CanTransferArrayBuffer(chunk.buffer) === false) {
```

Comment:
This check can easily be removed. We can move [these lines](https://github.com/whatwg/streams/blob/5da8d5c2bfdde4e3ce34ec3e63f4bd3f86922390/reference-implementation/lib/abstract-ops/readable-streams.js#L1010-L1013) in `ReadableByteStreamControllerEnqueue` to the start of that function, so that `TransferArrayBuffer` happens earlier (before we invalidate the BYOB request).